doc/api/cli.md

```diff
@@ -280,6 +280,10 @@ effort to report stack traces relative to the original source file.
 Overriding `Error.prepareStackTrace` prevents `--enable-source-maps` from
 modifying the stack trace.
 
+Note, Accessing `Error.stack` with source maps enabled can be an expensive 
```

Comment:
Updated the text to be less of a warning and more of a note on performance implication when accessing `Error.stack`. Let me know if this looks better than before.